src/codegen/compile.ml

```diff
@@ -1026,11 +1036,10 @@ module RTS = struct
     E.add_func_import env "rts" "stream_shutdown" [I32Type] [];
     E.add_func_import env "rts" "stream_reserve" [I32Type; I32Type] [I32Type];
     E.add_func_import env "rts" "stream_stable_dest" [I32Type; I64Type; I64Type] [];
-    E.add_func_import env "rts" "post_write_barrier" [I32Type] [];
-    E.add_func_import env "rts" "write_with_barrier" [I32Type; I32Type] [];
-    E.add_func_import env "rts" "allocation_barrier" [I32Type] [I32Type];
-    E.add_func_import env "rts" "stop_gc_on_upgrade" [] [];
-    E.add_func_import env "rts" "running_gc" [] [I32Type];
+    if !Flags.gc_strategy == Flags.Incremental then
```

Comment:
```suggestion
    if !Flags.gc_strategy = Flags.Incremental then
```
`==` is Ocaml's physical equality

Reply:
Thank you very much. I fixed these cases, including one that I had introduced in `https://github.com/dfinity/motoko/pull/3837`.